introduction.md

```diff
@@ -0,0 +1 @@
+edited this file because they wanted a sentence in it. here's a sentence. 

```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.